rfc.sh

```diff
@@ -6,14 +6,18 @@
 
 UPSTREAM=${GLUSTER_UPSTREAM}
 if [ "x$UPSTREAM" = "x" ]; then
-    for rmt in $(git remote); do
-	rmt_repo=$(git remote show $rmt -n | grep Fetch | awk '{ print $3 }');
```

Comment:
Why don't just use this:

    git remote get-url ${rmt}

Reply:
I used  `git config` to check all remotes at once.

Then, for a given remote, yes, this could work too.
The get-url action seems to have been added in git 2.7. The config regex search was introduced earlier. (If it matters and anyone is developping on eg RHEL7/CentOS7.)

@mykaul proposed to do a minimal change with the `LANGUAGE` viariable in front of the git remote call. I will push a change in that sense. Then, I guess the options are there.

Reply:
Ok, that's fine